framer/Material-UI.framerfx/code/Icon.tsx

```diff
@@ -7,6 +7,7 @@ import { pascalCase } from './utils';
 interface Props extends SvgIconProps {
   baseClassName: string;
   color: 'action' | 'disabled' | 'error' | 'inherit' | 'primary' | 'secondary';
+  sx?: object;
```

Comment:
Let's revert these changes. Take a look on this section https://gist.github.com/mnajdova/b0562785b7cf1b9d9aeae382d578594b#potential-issues-and-their-fixes for fixing the framer issue.

Reply:
Ah, hadn't seen that section, thanks!  (Need to read things more carefully.  :P)